### PR TITLE
refactor: drop modules inside addon-commerce

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

**Review by nsbarsukov — APPROVED** [review]

File: projects/addon-commerce/components/input-card-grouped/input-card-grouped.component.ts [hunk 1]
```diff
@@ -50,7 +64,26 @@ import {TUI_INPUT_CARD_GROUPED_TEXTS} from './input-card-grouped.providers';
 const EXPIRE_COMPLETE_LENGTH = 5; // MM/YY
 
 @Component({
+    standalone: true,
     selector: 'tui-input-card-grouped',
+    imports: [
+        NgIf,
+        AsyncPipe,
+        TuiDropdownOptionsDirective,
```
Comment (waterplea) [inline projects/addon-commerce/components/input-card-grouped/input-card-grouped.component.ts:72]: Maybe we should just got with dropdown module here?

File: projects/demo/src/modules/components/input-card-grouped/index.html [hunk 2]
```diff
@@ -12,42 +12,37 @@
         <tui-doc-example
             id="validation"
             heading="With validation"
-            [content]="example1"
-        >
-            <tui-input-card-grouped-example-1 />
-        </tui-doc-example>
+            [component]="1 | tuiComponent"
+            [content]="1 | tuiExample: 'html,ts'"
```
Comment (waterplea) [inline projects/demo/src/modules/components/input-card-grouped/index.html:16]: ngFor?

File: projects/addon-commerce/components/input-card/input-card.component.ts [hunk 3]
```diff
@@ -50,6 +73,20 @@ export class TuiInputCardComponent extends AbstractTuiInputCard<string> {
         return !!this.input && this.input.focused;
     }
 
+    public get inputMode(): TuiTextfieldHost['inputMode'] {
```
Comment (waterplea) [inline projects/addon-commerce/components/input-card/input-card.component.ts:76]: Why do we need all these changes here?

Comment (splincode) [reply projects/addon-commerce/components/input-card/input-card.component.ts:76]: in the parent class there are fields

File: projects/demo/src/modules/components/input-card-grouped/index.html (hunk 2 above)
Comment (splincode) [reply projects/demo/src/modules/components/input-card-grouped/index.html]: fixed

**Pull request merged**